Changelog

```diff
@@ -16,10 +16,23 @@ Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v5.0.0
     - FEATURE: Add option to name each qbittorrent instance (Chocobo1)
     - FEATURE: Add button for sending test email (Thomas Piccirello)
     - FEATURE: Allow torrents to override default share limit action (glassez)
+    - FEATURE: Use Start/Stop instead of Resume/Pause (thalieht) 5d1c2496063b41874e85af0145b153e006fc92d3
+    - FEATURE: Add the Popularity metric (Aliaksei Urbanski) f37d0c486c36e9f6c6e62378385e639feada3085
+    - FEATURE: Focus on Download button if torrent link retrieved from the clipboard (glassez) 7310eec74e6694342e9934ecc524df38d0064bb5
+    - FEATURE: Add ability to pause/resume entire BitTorrent session (glassez) 8ef7d3ec9a56e7445948c37a282629ca5ed551ab
+    - FEATURE: Add an option to set BitTorrent session shutdown timeout (glassez) 2b728b3bc0c525905ef4c6086bc45027eb2150fd
+    - FEATURE: Apply filename filter to subfolder names as well (glassez) d89f289f82fb6ff4305841c7d4c7899ffe12f896
```

Comment:
```suggestion
    - FEATURE: Apply "Excluded file names" to folder names as well (glassez) d89f289f82fb6ff4305841c7d4c7899ffe12f896
```

I thought the current description was referring to "Filter files..." in content tab.